frontend/tablet/src/screens/RoomDetailScreen/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
+
+import { Typography, Button } from '../../components'
+import theme from '../../components/theme'
+
+const Footer: React.FC = () => {
+  return (
+    <>
+      {/* TODO extract style */}
+      <View style={styles.quickReservation}>
```

Comment:
`quckReservation` is not a good name for this style. This will change in case the room is currently booked to "extend functionality". `quickActions` would be an appropriate name for this section.